frontend/apps/battleship/src/main.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { initErrorTracking, logPublicEnvs } from '@dapps-frontend/error-tracking';
+import { initErrorTracking, logPublicEnvs, InitializeGoogleAnalytics } from '@dapps-frontend/error-tracking';
 import { App } from './app';
 
+InitializeGoogleAnalytics()
```

Comment:
```suggestion
InitializeGoogleAnalytics();
```

Check your formatter settings, please. Some of lines lack semi-colon.